src/puptoo/app.py

```diff
@@ -105,10 +105,13 @@ def main():
             CONSUMER_WAIT_TIME.observe(now - start)
             start = now
             try:
-                if msg.headers('service') in ['advisor','compliance','malware-detection']:
-                    header = msg.headers('service')
-                    msg = json.loads(msg.value().decode("utf-8"))
-                    handle_message(msg, header)
+                logger.info(msg)
+                for header in msg.headers():
+                    if header[0] == 'service' and header[1] in ['advisor','compliance','malware-detection']:
+                        service = header[1]
+                        msg = json.loads(msg.value().decode("utf-8"))
+                        handle_message(msg, service)
+                        break
```

Comment:
Instead of iterating on each header value, you can convert tuple to dict object and use it directly.
example - 
```
>>> headers = dict((('service', b'advisor'),('xyz', b'pqr')))
{'service': b'advisor', 'xyz': b'pqr'}
```